snippets/fsh-snippets.json

```diff
@@ -114,13 +137,31 @@
 		],
 		"description": "Create a FSH Invariant"
 	},
-
 	"FSH RuleSet": {
-		"prefix": ["RS", "rs", "rule", "Rule"],
+		"prefix": [
+			"RS",
+			"rs",
+			"rule",
+			"Rule"
+		],
 		"body": [
 			"RuleSet: $1",
 			"* $0"
 		],
 		"description": "Create a FSH RuleSet"
+	},
+	"Slicing rules": {
+		"prefix": [
+			"^slicing"
+		],
+		"body": [
+			"^slicing.discriminator.type = ${1|#value,#exists,#pattern,#type,#profile|}",
```

Comment:
I know this is in the order it is listed in the ElementDefinition documentation.  But I wonder if we should put `#pattern` first, as I think that is the type we see used most often...  But I could go either way on this, since I can see the argument for listing in the same order as the doc.